app/views/components/details_content_link.scala.html

```diff
@@ -16,4 +16,4 @@
 
 @()(implicit messages: Messages)
 
-<a target="_blank" href="https://www.gov.uk/government/collections/uk-trade-tariff-volume-3-for-cds"> @messages("declaration.type.tariffUrlText")</a>
\ No newline at end of file
+<a class="govuk-link govuk-link--no-visited-state" target="_blank" href="https://www.gov.uk/government/collections/uk-trade-tariff-volume-3-for-cds"> @messages("declaration.type.tariffUrlText")</a>
```

Comment:
Since this component is using GDS, could you move it to `gds` package?

Reply:
That's a good idea.   However, to avoid having to change all the existing pages that reference this component, how about I "clone" it to the gds package (with the new gds classes) and then we delete this version when it's no longer needed?

Reply:
Good point.

Reply:
created new class components.gds.detailsContentLink